website/src/components/QueryPlan/QueryPlan.jsx

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {useState}from 'react';
+import ReactJson from 'react-json-pretty';
+
+import 'react-json-pretty/themes/monikai.css';
+import '../QueryPlan/queryPlan.css'
+
+const QueryPlan = ({data, isOpen, togglePlan}) => {
+
+    const queryPlanKeyStyle = {
+        background: isOpen && window.innerWidth > 1225 ? 'orange' : 'initial',
+      };
+    
+      const queryTitleStyle = {
+        background: isOpen && window.innerWidth < 1225 ? 'orange' : 'initial',
+      };
```

Comment:
We should use constant for hardcoded numbers like these or add a comment describing it - otherwise we do not know what it is

Reply:
There is an indentation issue with the second const here